### PR TITLE
Add 1.3.0 changelog

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.3.0 (February 6, 2020)
+
+### Enhancements
+
+ * Add new preference to disable application.properties value validation. See [#201](https://github.com/redhat-developer/vscode-quarkus/issues/201)
```
Comment (fbricon) [inline CHANGELOG.md]: Add new `quarkus.tools.validation.value.severity` preference ...

**Review by fbricon — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+ * Improve checking vscode-yaml extension. See [#195](https://github.com/redhat-developer/vscode-quarkus/issues/195)
```
Comment (fbricon) [inline CHANGELOG.md]: we should rather mention https://github.com/redhat-developer/vscode-quarkus/pull/189

**Comment by xorye** [comment]
@fbricon should I mentioned all of the new preferences by their names?

image

File: CHANGELOG.md (hunk 1 above)
Comment (xorye) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (xorye) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.3.0 (February 6, 2020)
+
+### Enhancements
+
+ * Add new `quarkus.tools.validation.value.severity` preference to disable application.properties value validation. See [#201](https://github.com/redhat-developer/vscode-quarkus/issues/201)
+ * Add Qute grammar for JSON, YAML and txt files. See [#194](https://github.com/redhat-developer/vscode-quarkus/pull/194)
```
Comment (fbricon) [inline CHANGELOG.md]: what about Qute HTML?

Comment (xorye) [reply CHANGELOG.md]: fixed

**Comment by fbricon** [comment]
> @fbricon should I mentioned all of the new preferences by their names?

nah, should be fine

**Pull request merged**